cpp/include/cudf/join.hpp

```diff
@@ -153,6 +158,7 @@ std::unique_ptr<cudf::table> left_join(
   std::vector<cudf::size_type> const& left_on,
   std::vector<cudf::size_type> const& right_on,
   std::vector<std::pair<cudf::size_type, cudf::size_type>> const& columns_in_common,
+  bool compare_nulls_equal = true,
```

Comment:
Use https://github.com/rapidsai/cudf/blob/branch-0.15/cpp/include/cudf/types.hpp#L130

Reply:
Oh, shoot. Yes, that's so much better. Will update.

Reply:
Now fixed to use `null_equality`.